onnx_tf/handlers/backend/dropout.py

```diff
@@ -17,11 +17,13 @@ def _common(cls, node, **kwargs):
     x = tensor_dict[node.inputs[0]]
     attrs = copy.deepcopy(node.attrs)
 
-    if cls.SINCE_VERSION < 7:
```

Comment:
@pluradj The original code flow seems more easy to follow. May we keep the original code and modify line 20 to be "if cls.SINCE_VERSION < 7 and attrs.pop("is_test, 0) == 0:" and Line 23 to be "elif cls.SINCE_VERSION < 12:"

Reply:
@winnietsang Thanks, updated!